sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Config/ServiceBusHostBuilderExtensionsTests.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using Azure.Messaging.ServiceBus;
```

Comment:
Should we consider explicitly testing the coalescing logic for the different ways to specify `autoComplete` ?

Reply:
I think this is something that will need discussion around what the behavior should be. There isn't really a great way to back compat this if we go to only one option.